backend/kernelCI_app/views/hardwareDetailsView.py

```diff
@@ -432,6 +519,18 @@ def sanitize_records(self, records, trees: List, is_all_selected: bool):
 
         return (builds, tests, boots, tree_status_summary)
 
+    def _assign_default_record_values(self, record):
```

Comment:
```suggestion
    def _assign_default_record_values(self, record: Dict[str, Any]) -> None:
```

Reply:
There hasn't been any occurrence of `Any` in the project yet, I believe just `Dict` is the correct way here

Reply:
Yes, just Dict might be better considering the way we are typing the project up until this moment